verification/VerifyRNAWithUmis.wdl

```diff
@@ -0,0 +1,75 @@
+version 1.0
+
+import "../verification/VerifyMetrics.wdl" as MetricsVerification
+import "../verification/VerifyTasks.wdl" as VerifyTasks
+
+workflow VerifyRNAWithUMIs {
```

Comment:
Is this the same as the "VerifyRNAWithUMIs" that is currently checked in to develop?